### ISSUE TITLE
Environment variables ignored in production build

### ISSUE THREAD

**Issue by searsaw** [relax/relax#407]
If I run `PORT=3000 yarn start`, it still listens on port 8080. I tried passing the port also at build time by running `PORT=3000 yarn build`, but it still ignores it. It always tries to mount at port 8080. The only thing that allowed me to change the port in production was by creating a `.relaxrc` file and changing the port in there.

**Comment by bruno12mota** [comment]
Strange, should work. From config.js it should fetch the default port from env if available

**Comment by searsaw** [comment]
Yeah that's how I thought it would work too, but it doesn't seem to be doing that.
